timezone/test/Timezone_test.rb

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+Yast.import "Timezone"
+
+describe Yast::Timezone do
+  subject { Yast::Timezone }
+
+  describe "#ProposeLocaltime" do
+    it "returns true if a Windows partition is found" do
+      subject.windows_partition = true
+      expect(Yast::SCR).to receive(:Read)
+        .with(Yast::Path.new(".probe.is_vmware")).and_return(false)
```

Comment:
NP: I'd use ```allow``` instead of ```expect``` to mock the the SCR call. Just a semantic puristic issue :-)